client/login/controller.js

```diff
@@ -14,8 +15,31 @@ const enhanceContextWithLogin = ( context ) => {
 		params: { flow, isJetpack, isGutenboarding, socialService, twoFactorAuthType },
 		path,
 		query,
+		isServerSide,
 	} = context;
 
+	// Process a social login handoff from /start/user.
+	if ( query?.email_address && query?.service && query?.access_token && query?.id_token ) {
+		context.store.dispatch( {
+			type: SOCIAL_HANDOFF_CONNECT_ACCOUNT,
```

Comment:
Nice! Does this reducer automatically connect the social account when this data exists?

Reply:
> Nice! Does this reducer automatically connect the social account when this data exists?

@noahtallen It doesn't automatically connect, but takes them to this state:

![image](https://user-images.githubusercontent.com/36432/182207320-7df812be-5e55-4947-a819-6213df5302aa.png)

Reply:
Gotcha. What does the auth info help us do in this case? The way I read the original code, it seemed like we were putting the google auth info into the reducer. I think seeing "social" in the action type made me think that :)

Reply:
> What does the auth info help us do in this case?

@noahtallen The `access_token` and `id_token` are required to complete the connection process in the form above.

> The way I read the original code, it seemed like we were putting the google auth info into the reducer. I think seeing "social" in the action type made me think that :)

I'm not sure I follow... could you explain further?

Reply:
Ah, I think I misunderstood your initial comment -- it doesn't automatically connect at first, but it does still connect the email/password account to the Google account after you submit the form. That makes a lot of sense and matches what I'd expect as well.

When I read your initial comment, I thought that it wouldn't connect the Google account after logging in with email/pass.